ios/Helio.xcodeproj/xcshareddata/xcschemes/HelioStaging.xcscheme

```diff
@@ -79,14 +78,14 @@
             ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
             <ActionContent
                title = "Run Script"
-               scriptText = "echo &quot;.env.development&quot; &gt; /tmp/envfile&#10;">
+               scriptText = "# Type a script or drag a script file from your workspace to insert its path.&#10;echo &quot;.env.development&quot; &gt; /tmp/envfile&#10;">
                <EnvironmentBuildable>
                   <BuildableReference
                      BuildableIdentifier = "primary"
                      BlueprintIdentifier = "888DAA1429B48C8900311304"
-                     BuildableName = "BetterSocialStaging.app"
-                     BlueprintName = "BetterSocialStaging"
-                     ReferencedContainer = "container:BetterSocial.xcodeproj">
+                     BuildableName = "HelioStaging.app"
+                     BlueprintName = "HelioStaging"
+                     ReferencedContainer = "container:Helio.xcodeproj">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The script text has been updated with a comment. Ensure that this comment is accurate and helpful for future developers. Also, the app name and container reference have been updated here as well.
```diff
-   scriptText = "echo &quot;.env.development&quot; &gt; /tmp/envfile&#10;"
-   BuildableName = "BetterSocialStaging.app"
-   BlueprintName = "BetterSocialStaging"
-   ReferencedContainer = "container:BetterSocial.xcodeproj"
+   scriptText = "# Type a script or drag a script file from your workspace to insert its path.&#10;echo &quot;.env.development&quot; &gt; /tmp/envfile&#10;"
+   BuildableName = "HelioStaging.app"
+   BlueprintName = "HelioStaging"
+   ReferencedContainer = "container:Helio.xcodeproj"
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->